README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/expectedbehavior/acts_as_archival.svg?branch=master)](https://travis-ci.org/expectedbehavior/acts_as_archival)
```

Comment:
It's up to you, but there's two I'd personally use. The one by [Gemnasium] regarding out of date dependencies (although I am not sure how well it plays with Appraisal) and wrt RuboCop, yes builds will fail, but if you want an automatic review, [Hound CI](https://houndci.com/configuration#ruby) can do that.

Maybe this one, to easily get an entry for your Gemfile.
[![Gem Version](https://badge.fury.io/rb/acts_as_archival.svg)](https://badge.fury.io/rb/acts_as_archival)